common.go

```diff
@@ -453,9 +453,11 @@ func GetLanguageByAlias(alias string) (lang string, ok bool) {
 // GetLanguagesByGitAttributes returns either a string slice with the lenguage if the filename match with a regExp in .gitattributes
 //or return nil in case of none regexp matchs the filename . It complies with the signature to be a Strategy type.
```

Comment:
* space before _or_
* s/return/returns/
* no space after _filename_
* s/in case of none/in case no/
* s/matchs/matches/